### DISCUSSION TITLE
Negated Linked PR search filter not working as expected

### DISCUSSION THREAD

**Discussion by ezeholz** [community/community#126556] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

I found an issue with the filtered search. When I try to filter by `-linked:pr`, there's an issue that shows up in that search, even through it has a PR linked
![image](https://github.com/community/community/assets/30635862/1cf5c6ec-226f-4fd4-ad68-840ba1dd5da9)

Might be because it has some issues linked, so it's overwriting the actual linked PR.
![image](https://github.com/community/community/assets/30635862/892d8a2f-b703-40d9-9bf1-408ac3df56c9)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Viktor-Ivliev** [comment]
also problem: https://github.com/orgs/community/discussions/125226
